org.scala-ide.sdt.core/src/org/scalaide/ui/internal/jdt/model/ScalaOverrideIndicatorBuilder.scala

```diff
@@ -35,28 +39,68 @@ case class JavaIndicator(scu: ScalaCompilationUnit,
   }
 }
 
-trait ScalaOverrideIndicatorBuilder { self : ScalaPresentationCompiler =>
```

Comment:
Yes, I don't know either. I think the problem was the constructor parameter `base: Symbol`. That type is nested inside the compiler.